js/foundation.sticky.js

```diff
@@ -38,7 +38,13 @@ class Sticky {
     if (!$parent.length) {
       this.wasWrapped = true;
     }
-    this.$container = $parent.length ? $parent : $(this.options.container).wrapInner(this.$element);
+    if($parent.lenth){
+        this.$container = $parent;
```

Comment:
@randmanrjr Still need a bit more indentation fix 
=>  If confused, compare line 39 and 42 !

{{ Yes i am annoying :smile: }}